.azure/autotest_template.yml

```diff
@@ -16,9 +16,9 @@ jobs:
     displayName: Initialize MAVLink submodule
   - script: git submodule update --recursive --init modules/uavcan
     displayName: 'Initialize UAVCAN submodule'
-  - script: C:\Cygwin\bin\bash --login -c "cd $(cygpath '%BUILD_SOURCESDIRECTORY%')/modules/mavlink/pymavlink && python setup.py build install --user"
+  - script: C:\Cygwin\bin\bash --login -c "cd $(cygpath '%BUILD_SOURCESDIRECTORY%')/modules/mavlink/pymavlink && python3 setup.py build install --user"
```

Comment:
This change is wrong, there is no python3 executable, which is why someone added the links you forced below. This is breaking the nightly tests.

Reply:
I was pretty sure that the log show that python3  was there... Anyways, I will revert that. Thanks